api/controllers/game/ready.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (req, res) {
         }
         if (bothReady) {
           // Inform all clients this game is starting
-          sails.sockets.blast('gameStarting', { gameId: game.id });
+          sails.sockets.blast('gameStarted', { gameId: game.id });
```

Comment:
Make sure anything using this identifier is aware of this new name. It think these ids should live in a constants file we share eventually so we can make sure it's always correct, something like

```
const SOCKET_EVENT_GAME_STARTED = 'gameStarted';
```

Reply:
Yeah I think that's worth making it's own issue